### PR TITLE
Unittest

### PR DESCRIPTION
**Pull request by Incorrect-password**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -2,9 +2,9 @@
 /**
  * sets up a connection to the collection database
  *
- * @return array variable containing the database info
+ * @return object object containing the database info
```
Comment (jasonrees1) [inline functions.php]: object object should be object $db

File: functions.php [hunk 2]
```diff
@@ -16,7 +16,7 @@ function collectionDbConnection()
  * @param $db - all the data from the database
```
Comment (jasonrees1) [inline functions.php:16]: Should this be object $db as in line 5

File: functions.php [hunk 3]
```diff
@@ -16,7 +16,7 @@ function collectionDbConnection()
...
  * @return array an array containing all of the data from the specified fields.
```
Comment (jasonrees1) [inline functions.php]: should this have array $allItems

File: functions.php [hunk 4]
```diff
@@ -28,7 +28,7 @@ function retrieveData($db)
  * @param $allItems the chosen data selected from a database
```
Comment (jasonrees1) [inline functions.php:28]: same as above

File: test/functions.php [hunk 5]
```diff
@@ -6,5 +6,11 @@
...
 {
-    public function testCollectionDbConnection
-}
\ No newline at end of file
+
+    public function testDisplayData()
+    {
+        $cars = [['make'=>"vw"],['make'=>'bmw']];
+        $result = displayData($cars);
+        $this->assertTrue(is_string($result));
```
Comment (jasonrees1) [inline test/functions.php:14]: maybe test the contents of the result as well as the data type

**Review by jasonrees1 — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (Incorrect-password) [reply functions.php]: I have changed to object $db

File: functions.php (hunk 3 above)
Comment (Incorrect-password) [reply functions.php]: changed

File: test/functions.php (hunk 5 above)
Comment (Incorrect-password) [reply test/functions.php:14]: added the below assertion

**Review by jasonrees1 — APPROVED** [review]

**Pull request merged**